packages/edit-site/src/components/header/index.js

```diff
@@ -62,7 +62,7 @@ export default function Header() {
 			</div>
 			<div className="edit-site-header__actions">
 				<SaveButton />
-				<MoreMenu />
```

Comment:
Was this removal intentional or did it just end up there during the rebase?

Reply:
Probably a rebase issue.

Reply:
> Probably a rebase issue.

Ok, I'll spin up a quick PR to bring it back.